### PR TITLE
Integrate congo console from goconserver with rcons

### PR DESCRIPTION
**Pull request by chenglch**
Enhance the original implement of rcons to support goconserver.
`rcons` will help the user choose from one of the console backend
service based on the console server status of management node.

Implement #4216

After this pull request is merged, goconserver could be tested with xcat.

Some simple steps to setup and try `goconserver` on ppc64le platforms.
```
wget https://github.com/chenglch/goconserver/files/1474108/goconserver_linux_ppc64le.tar.gz
tar xvfz  goconserver_linux_ppc64le.tar.gz
cd goconserver_linux_ppc64le
./setup.sh
service conserver stop
makegocons <node>
rcons <node>
```

### CODE REVIEW

**Comment by chenglch** [comment]
After this pull request is merged, goconserver could be tested with xcat.

Some simple steps to setup and try `goconserver` on ppc64le platforms.
```
wget https://github.com/chenglch/goconserver/files/1474108/goconserver_linux_ppc64le.tar.gz
tar xvfz  goconserver_linux_ppc64le.tar.gz
cd goconserver_linux_ppc64le
./setup.sh
service conserver stop
makegocons <node>
rcons <node>
```

File: xCAT-client/bin/rcons [hunk 1]
```diff
@@ -103,6 +125,38 @@ if [ $USE_CONFLUENT == "1" ] && ([ -x "/opt/confluent/bin/confetty" ] || [ -x "/
...
+    if [ -z "$CONSERVER" ]; then
+        CONSERVER=$XCATHOST
+    fi
+    if [ -z "$CONSERVER" ]; then
+        CONSERVER=`hostname`
+    fi
+    CONGO_ENV="CONGO_SSL_KEY=$HOME/.xcat/client-cred.pem \
+               CONGO_SSL_CERT=$HOME/.xcat/client-cred.pem \
+               CONGO_SSL_CA_CERT=$HOME/.xcat/ca.pem \
+               CONGO_URL=https://$CONSERVER:12429 \
```
Comment (zet809) [inline xCAT-client/bin/rcons]: The 12429 is for configuration, why is also used for congo?

Comment (chenglch) [reply xCAT-client/bin/rcons]: You are right, I have changed it to `CONGO_PORT=12430` which is used to set the console port.

**Comment by zet809** [comment]
Looks good to me, I agree to merge, thx @chenglch

**Pull request merged**